source/docs/articles/sites/varnish/caching-advancedtopics.md

```diff
@@ -142,7 +142,7 @@ If you're using the Environment Access: Locked security setting on a site envir
 
 ## Pantheon's .vcl File
```

Comment:
@bmackinney Ah, that's a bummer. How about creating a separate issue for that? I think there's more value in having an accurate section heading than redirecting to a specific heading. As long as people land on the right page, I think they should be able to find what they need. What do you think?